MultiVersions/Forge/main/beachparty/lang/zh_tw.json

```
@@ -0,0 +1,143 @@
+{
+  "beachparty.container.cabinet": "櫥櫃",
+  "block.beachparty.beach_chair":"沙灘椅",
+  "block.beachparty.beach_grass": "沙灘草",
+  "block.beachparty.beach_towel": "沙灘巾",
+  "block.beachparty.cabinet": "熱帶風格櫥櫃",
+  "block.beachparty.chair":"棕櫚木椅",
+  "block.beachparty.cocoa_cocktail": "可可豆雞尾酒",
+  "block.beachparty.coconut": "椰子",
+  "block.beachparty.coconut_block": "椰子",
+  "block.beachparty.coconut_cocktail": "椰子雞尾酒",
+  "block.beachparty.deck_chair": "棕櫚木躺椅",
+  "block.beachparty.dried_wheat_block": "乾小麥捆",
+  "block.beachparty.dried_wheat_slab": "乾小麥捆半磚",
+  "block.beachparty.dried_wheat_stairs": "乾小麥捆階梯",
+  "block.beachparty.dry_bush":"乾灌木",
+  "block.beachparty.dry_bush_tall": "高乾灌木",
+  "block.beachparty.empty_sand_bucket": "空沙桶",
+  "block.beachparty.empty_sand_bucket_block": "空沙桶",
+  "block.beachparty.hammock": "沙灘吊床",
+  "block.beachparty.honey_cocktail": "蜂蜜雞尾酒",
+  "block.beachparty.lounge_chair":"棕櫚木躺椅",
+  "block.beachparty.melon_cocktail": "西瓜雞尾酒",
+  "block.beachparty.message_in_a_bottle": "瓶中信",
+  "block.beachparty.mini_fridge": "迷你冰箱",
+  "block.beachparty.notimplemented" : "此方塊的功能尚未完全實現。敬請期待！",
+  "block.beachparty.palm_beam": "棕櫚木樑",
+  "block.beachparty.palm_button": "棕櫚木按鈕",
+  "block.beachparty.palm_door": "棕櫚木門",
+  "block.beachparty.hanging_sign": "懸掛式棕櫚木告示牌",
+  "block.beachparty.palm_fence": "棕櫚木柵欄",
+  "block.beachparty.palm_fence_gate": "棕櫚木柵欄門",
+  "block.beachparty.palm_floorboard": "棕櫚木地板",
+  "block.beachparty.palm_leaves": "棕櫚木樹葉",
+  "block.beachparty.palm_log": "棕櫚木原木",
+  "block.beachparty.palm_planks": "棕櫚木材",
+  "block.beachparty.palm_pressure_plate": "棕櫚木壓力板",
+  "block.beachparty.palm_sapling": "棕櫚木樹苗",
+  "block.beachparty.palm_sign": "棕櫚木告示牌",
+  "block.beachparty.palm_slab": "棕櫚木半磚",
+  "block.beachparty.palm_stairs": "棕櫚木階梯",
+  "block.beachparty.palm_tall_torch": "棕櫚木高火把",
+  "block.beachparty.palm_torch": "棕櫚木火把",
+  "block.beachparty.palm_torch_item": "棕櫚木火把",
+  "block.beachparty.palm_trapdoor": "棕櫚木地板門",
+  "block.beachparty.palm_wall_torch": "牆上的棕櫚木火把",
+  "block.beachparty.palm_wood": "棕櫚木塊",
+  "block.beachparty.pumpkin_cocktail": "南瓜雞尾酒",
+  "block.beachparty.radio": "收音機",
+  "block.beachparty.sand_bucket": "沙桶",
+  "block.beachparty.sand_bucket_block": "沙桶",
+  "block.beachparty.sand_dirty": "帶雜物的沙",
+  "block.beachparty.sand_pile": "沙堆",
+  "block.beachparty.sand_seastars": "帶海星的沙",
+  "block.beachparty.sand_slab": "沙半磚",
+  "block.beachparty.sandcastle": "沙堆城堡",
+  "block.beachparty.sandwaves": "海浪沙",
```

用來形容沙的外觀的話，我感覺波浪比較合適
```suggestion
  "block.beachparty.sandwaves": "波浪沙",
```